packages/sdk/src/oracle/services/optimisticOracle.ts

```diff
@@ -1,42 +1,14 @@
 import { optimisticOracle } from "../../clients";
 import { BigNumberish, Provider, Signer, TransactionResponse, Log, TransactionReceipt } from "../types/ethers";
 import type { OracleInterface, RequestKey, OracleProps, Request } from "../types/interfaces";
-import { requestId, insertOrderedAscending, eventKey, isUnique, getAddress } from "../utils";
+import { requestId, insertOrderedAscending, eventKey, isUnique } from "../utils";
 
 import { RequestPrice, ProposePrice, DisputePrice, Settle } from "../../clients/optimisticOracle";
 
 export type OptimisticOracleEvent = RequestPrice | ProposePrice | DisputePrice | Settle;
 
 export type { RequestPrice, ProposePrice, DisputePrice, Settle };
 
-// this had to be copied in because interfaces in contracts-frontend and contracts-node are different
```

Comment:
removing all these lookups, its just way too dangerous because reviewers will typically not see the addresses being used. the frontend client must be explicit now when providing oracle addresses so that we can have more checks on what addresses are used

Reply:
@smb2796 do you know if we have any third-party teams using the SDK who need to be aware of this change, or is the SDK only for our own interfaces so far?

Reply:
afaik only we are using this but def check around

Reply:
we can also signal this as a breaking change